src/content/en/fundamentals/native-hardware/capturing-images/index.md

```diff
@@ -2,116 +2,242 @@ project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 description: Most browsers can get access to the user's camera.
 
-{# wf_updated_on: 2016-08-22 #}
+{# wf_updated_on: 2017-07-28 #}
 {# wf_published_on: 2016-08-23 #}
 
 # Capturing an Image from the User {: .page-title }
 
 {% include "web/_shared/contributors/paulkinlan.html" %}
+{% include "web/_shared/contributors/mscales.html" %}
 
-Many browsers now have the ability to access video and audio input from the 
-user. However, depending on the browser it might be a full dynamic and inline 
-experience, or it could be delegated to another app on the user's device.
+Many browsers now have the ability to access video and audio input from the
+user. However, depending on the browser it might be a full dynamic and inline
+experience, or it could be delegated to another app on the user's device. On top
+of that, not every device even has a camera. So how can you create an experience
+that uses a user generated image that works well everywhere?
 
 ## Start simple and progressively
 
 The easiest thing to do is simply ask the user for a pre-recorded
-file. Do this by creating a simple file input element and adding 
-an `accept` filter that indicates we can only accept image files and ideally we 
-will get them directly from the camera.
+file. There are a lot of good options here to give you a great progressive enhancement story.
 
-    <input type="file" accept="image/*" capture>
+### Ask for a URL
+This is the best supported but least satisfying option - get the user to give you a URL, and then
+use that. For just displaying the image this works everywhere - create an `img` element, set the
+`src` and you're done.
+
+It's more complicated than that, though, if you want to manipulate the image in any way. CORS will
+prevent you from accessing the actual pixels unless the server sets the appropriate headers, and the
```

Comment:
And you also have to define `crossOrigin`: https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image